src/lib-settings/settings-parser.h

```diff
@@ -179,9 +179,9 @@ bool settings_check(const struct setting_parser_info *info, pool_t pool,
 void settings_parse_set_expanded(struct setting_parser_context *ctx,
 				 bool is_expanded);
 /* Mark all the parsed settings with given keys as being already expanded. */
-void settings_parse_set_key_expandeded(struct setting_parser_context *ctx,
+void settings_parse_set_key_expanded(struct setting_parser_context *ctx,
 				       pool_t pool, const char *key);
-void settings_parse_set_keys_expandeded(struct setting_parser_context *ctx,
+void settings_parse_set_keys_expanded(struct setting_parser_context *ctx,
 					pool_t pool, const char *const *keys);
```

Comment:
This changes the API. For v2.2 need to verify that these aren't used outside Dovecot core. A quick grep didn't show anything. In any case this should be in its own commit.